README.md

```diff
@@ -64,6 +64,14 @@ on how vmagent is configured to scrape the app server._
 
 **Required.** The basic auth password for the remote write URL.
 
+#### `VMAGENT_SCRAPE_USERNAME`
```

Comment:
Let's add these to the example commands in `Usage` too.

Reply:
Good idea - updated 👍